README.md

```diff
@@ -44,6 +44,7 @@ This library currently supports the following cryptocurrencies and address forma
  - ATOM (bech32)
  - TRX (base58check)
  - NEM (base32)
+ - NEO (base58check)
```

Comment:
Can you provide reference link so that we can verify?

Reply:
@makoto  https://docs.neo.org/v3/docs/en-us/tooldev/concept/cryptography/encode_algorithm.html